### ISSUE TITLE
Problem about normalization

### ISSUE THREAD

**Issue by liya2001** [qijiezhao/s3d.pytorch#2]
Hi @qijiezhao , thank you for sharing your implementation. I am wondering if  your pre-trained RGB_imagenet.pkl is just converted from deepmind's tf model weights or not. According to I3D's repository, pixels should be scaled between -1 and 1, but I can't find this normalization step in your code, is this a negligence or for other reasons, thanks again.
